e2e/scripts/st_plotly_chart.py

```diff
@@ -121,8 +123,10 @@
     )
 )
 
-fig_waterfall.update_layout(title="Profit and loss statement 2018", showlegend=True)
-st.plotly_chart(fig_waterfall, theme="streamlit")
+fig_waterfall.update_layout(
+    title="Profit and loss statement 2018", height=300, width=300, showlegend=True
+)
+st.plotly_chart(fig_waterfall, use_container_width=False, theme="streamlit")
```

Comment:
Checks for `use_container_width=False` when height and width are changed.

Reply:
It seems like, if you're testing that the container width will be used/ignored in various conditions, you shouldn't be setting `use_container_width` explicitly.